cmd/run/run.go

```diff
@@ -425,9 +429,49 @@ func (c *Command) forkChild() (pid int, sec *seccomp.Listener, err error) {
 	if err != nil {
 		return 0, nil, fmt.Errorf("get executable: %w", err)
 	}
+
+	outfd := 1
+	errfd := 2
+
+	if c.flags.tracelogs {
+		outpipe := make([]int, 2)
```

Comment:
always use fixed size arrays when interacting with the kernel, Go's `int` is variable sized based on the architecture (not that we support 32-bit machines, but explicit is better and easier to debug later)